test/600-bridge-vrf.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# bridge driver tests with vrf
+#
+
+load helpers
+
+@test vrf - bridge with vrf {
+    run_in_host_netns ip link add test-vrf type vrf table 10
+    run_in_host_netns ip link set dev test-vrf up
+
+    run_netavark --file ${TESTSDIR}/testfiles/simplebridge-vrf.json setup $(get_container_netns_path)
+    result="$output"
+    assert_json "$result" 'has("podman")' == "true" "object key exists"
+
+    mac=$(jq -r '.podman.interfaces.eth0.mac_address' <<<"$result")
+    # check that interface exists
+    run_in_container_netns ip -j --details link show eth0
+
+    # check if vrf exists
+    run_in_host_netns ip -j --details link show podman0
+    result="$output"
+    assert_json "$result" ".[].linkinfo.info_slave_kind" "==" "vrf" "Bridge has a vrf set"
+}
+
+@test vrf - simple bridge {
+    run_netavark --file ${TESTSDIR}/testfiles/simplebridge.json setup $(get_container_netns_path)
+    run_in_host_netns ip -j --details link show podman0
+    result="$output"
+    assert_json "$result" ".[].linkinfo.info_slave_kind" "==" "null" "VRF is not set"
+}
+
+@test vrf - non existent vrf {
+    expected_rc=1 run_netavark --file ${TESTSDIR}/testfiles/simplebridge-vrf.json setup $(get_container_netns_path)
```

Comment:
IF you check for errors please also check for the errors message

Reply:
I added a check to verify if we receive the correct error message.